java/test/src/main/java/io/ray/test/NodeIpTest.java

```diff
@@ -0,0 +1,39 @@
+package io.ray.test;
+
+import io.ray.api.Ray;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = {"cluster"})
+public class NodeIpTest extends BaseTest{
+
+  private static final String NODE_IP = "127.0.0.2";
+
+  @BeforeClass
+  public void setUp() {
+    System.setProperty("ray.head-args.0", "--node-ip-address=127.0.0.2");
+    System.setProperty("ray.node-ip", "127.0.0.2");
```

Comment:
I'm wondering that whether we need to specify the node IP manually when we start a Python driver and connect to an existing node other than 127.0.0.1?

Reply:
Yes, with Python code, we also need to specify the node IP.

Reply:
By using `ray.init(_node_ip_address=xxx)`? It looks like a non-public API to me.

Reply:
Yes, there are multiple NICs/IPs in our environments. We need to set the specified to use.